pinax/stripe/webhooks.py

```diff
@@ -255,7 +255,12 @@ class CustomerUpdatedWebhook(Webhook):
     description = "Occurs whenever any property of a customer changes."
 
     def process_webhook(self):
-        customers.sync_customer(self.event.customer)
+        cu = None
+        try:
+            cu = self.event.message['data']['object']
```

Comment:
double-quotes please :)

Reply:
Sure.  Anything apart from that?
Shouldn't there be a test (change)?

Reply:
Also: maybe we could have pylint or something similar to report this?

Reply:
Let machines work for us.. https://github.com/pinax/pinax-stripe/pull/372